README.md

```diff
@@ -10,150 +10,148 @@
+You're all set!
 
-### Configuration Notes in High Availability Mode
+## Using the Plugin
 
-If you are running Mattermost v5.11 or earlier in [High Availability mode](https://docs.mattermost.com/deployment/cluster.html), please review the following:
+Once configuration is complete, run the `/github connect` slash command from any channel within Mattermost to connect your Mattermost account with GitHub.
 
-1. To install the plugin, [use these documented steps](https://docs.mattermost.com/administration/plugins.html#plugin-uploads-in-high-availability-mode)
-2. Then, modify the config.json [using the standard doc steps](https://docs.mattermost.com/deployment/cluster.html#updating-configuration-changes-while-operating-continuously) to the following
+## Onboarding Your Users
 
-```
-"PluginSettings": {
-    ...
-    "Plugins": {
-        "github": {
-            "encryptionkey": "<your encryption key, from step 5 above>",
-            "githuboauthclientid": "<your oauth client id, from step 2 above>",
-            "githuboauthclientsecret": "<your oauth client secret, from step 2 above>",
-            "githuborg": "<your github org>",
-            "webhooksecret": "<your webhook secret, from step 3 above>"
-        },
-    ...
-    "PluginStates": {
-        ...
-        "github": {
-            "Enable": true
-        },
-        ...
-    }
-},
-```
+When you’ve tested the plugin and confirmed it’s working, notify your team so they can connect their GitHub account to Mattermost and get started. Copy and paste the text below, edit it to suit your requirements, and send it out.
 
-## Development
+> Hi team, 
 
-This plugin contains both a server and web app portion.
+> We've set up the Mattermost GitHub plugin, so you can get notifications from GitHub in Mattermost. To get started, run the `/github connect` slash command from any channel within Mattermost to connect your Mattermost account with GitHub. Then, take a look at the [slash commands](https://github.com/mattermost/mattermost-plugin-github#slash-commands) section for details about how to use the plugin.
 
-- Use `make dist` to build distributions of the plugin that you can upload to a Mattermost server.
-- Use `make check-style` to check the style.
-- Use `make deploy` to deploy the plugin to your local server. 
+## Slash Commands
 
-Before running `make deploy` you need to set a few environment variables:
+* __Autocomplete slash commands__ - Explore all the available slash commands by typing `/` in the text input box - the autocomplete suggestions help by providing a format example in black text and a short description of the slash command in grey text. Visit the [executing commands](https://docs.mattermost.com/help/messaging/executing-commands.html) documentation for more details. You can also create custom slash commands. Visit the [slash commands](https://docs.mattermost.com/developer/slash-commands.html) documentation for more information and a list of default slash commands.
```

Comment:
1/5 Mentioning custom slash command seems a bit weird to me. The audience of this documents are admins who are setting up the GitHub plugin, not developers of slash commands.